### ISSUE TITLE
(possibly) Conflicting definitions for ScriptResult type 

### ISSUE THREAD

**Issue by juliandescottes** [w3c/webdriver-bidi#217]
I found two "definitions" for script's `ScriptResult`.
One at 7.3.1 https://w3c.github.io/webdriver-bidi/#module-script-definition

```
ScriptResult = (
  ScriptCallFunctionResult //
  ScriptEvaluateResult //
  ScriptGetRealmsResult
)
```

and another at 7.3.2.6 https://w3c.github.io/webdriver-bidi/#type-script-Result

```
ScriptResult = (
  ScriptResultSuccess //
  ScriptResultException
)

ScriptResultSuccess = {
  result: RemoteValue,
  realm: Realm
}

ScriptResultException = {
  exceptionDetails: ExceptionDetails
  realm: Realm
}
```

I am not sure if they refer to different things, but the fact that they use the same name feels confusing?

I can see that most modules have a "{ModuleName}Result" defined in the "definition" section, so from that point of view, having "ScriptResult" defined in 7.3.1 https://w3c.github.io/webdriver-bidi/#module-script-definition seems consistent.
Maybe 7.3.2.6 https://w3c.github.io/webdriver-bidi/#type-script-Result be renamed to avoid reusing the same name?

**Comment by juliandescottes** [comment]
cc @whimboo, topic we discussed earlier

**Comment by whimboo** [comment]
@jgraham and @sadym-chromium looks like there is another issue with a non detected broken CDDL type (issue #223 could help here as well).

**Comment by sadym-chromium** [comment]
WIP

**Comment by sadym-chromium** [comment]
Fixed in #224 

@juliandescottes cannot assign you as a reviewer for whatever reason (

**Comment by juliandescottes** [comment]
> Fix in #224
> 
> @juliandescottes cannot assign you as a reviewer for whatever reason (

Thanks for the PR! Not sure why I can't be assigned as a reviewer, I probably don't have the appropriate rights on the repo. But I "can" review the PR even if I'm not tagged, there's still that.

**Comment by juliandescottes** [comment]
https://github.com/w3c/webdriver-bidi/pull/224 was merge, we can close this.
